README.md

```diff
@@ -35,6 +35,8 @@ Loading cached images from config file.
 $ kubectl create configmap -n kube-system kubevirt-config --from-literal debug.useEmulation=true
 ```
 
+Deploying KubeVirt on `minishift`, you will need [to install openshift-client-tools and add the following SCCs](#running-on-okd-or-minishift) prior kubevirt.yaml deployment:
+
```

Comment:
I think that this fits better in the first paragraph of "Deploy KubeVirt".

If not, then please take a look at the guides below for [minikube](#setting-up-minikube) and [minishift](#running-on-okd-or-minishift)

Reply:
Actually - A good place could be to directly add it to the list right at the top of the document I.e.:

```
* minikube …
* minishift with OKD 3.11+ (See notes [here](link-to-notes-at-the-doc-end))
```

Reply:
so why have you accept it neither in "my" location nor yours?

Reply:
Because now it should catch the readers eyes when he is just following the first list of steps he sees when looking at the readme.

Reply:
I don't like it, because attention does not come free. minishift is just one perquisite, not the core issue here. Never mind, no biggie.